src/angularjs/src/app/index.route.js

```diff
@@ -20,7 +20,7 @@
                 template: '<ui-view/>'
             })
             .state('places.list', {
-                url: '',
+                url: ':place1/:place2/:place3/',
```

Comment:
Would this be necessary if we could add compare cities from the compare page?  I see how it's doing something useful, but it seems awkward to me.  I.e. it's user-visible and I think to a user it would not be clear why the places list should have such a long, complicated URL.